app/experimenter/experiments/api/v5/serializers.py

```diff
@@ -553,18 +561,20 @@ class Meta:
 
     def validate_reference_branch(self, value):
         if value["description"] == "":
-            raise serializers.ValidationError("Description cannot be blank.")
+            raise serializers.ValidationError(
+                {"description": ["This field cannot be blank."]}
```

Comment:
This is unrelated to your PR but a teeny win you could get in at least here - I met with Ana earlier to discuss review messaging and since the default error messages use "may not" instead of "cannot," that's the nomenclature we went with on the risk mitigation questions. This should probably be "This field may not be blank." to match the other generic blank field messages.

It may also be worth adding this to the [`NimbusConstants` serializer error messages](https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L500).